Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -340,6 +340,29 @@ class ProductStoreTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    /// Verifies that `ProductAction.retrieveProduct` returns the expected `Product` of external product type.
+    ///
+    func testRetrieveSingleExternalProductReturnsExpectedFields() {
+        // Arrange
+        let expectation = self.expectation(description: "Retrieve single external product")
+        let remote = MockProductsRemote()
+        let expectedProduct = MockProduct().product(siteID: sampleSiteID, productID: sampleProductID, buttonText: "Deal", externalURL: "https://goodeals.com")
```

Comment:
The website seems an existing website with an old SSL certificate. I suggest using `https://example.com` if possible.

Reply:
I totally made this URL up and never went to that URL 😅 to avoid confusion, I updated this URL in https://github.com/woocommerce/woocommerce-ios/pull/2392/commits/6dc3ae7dfe6bddc991c9236c9009be0b7275ca58